src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -211,6 +211,10 @@ internal static PackageSpec GetPackageSpec(JsonTextReader jsonReader, string nam
                             }
                         }
                         break;
+
+                    case "SDKAnalysisLevel":
```

Comment:
reads `SDKAnalysisLevel`  value from the dgspec file so that restorecommand can pass it to restore runner